api/pom.xml

```diff
@@ -235,6 +235,11 @@
 		   <groupId>net.sf.ehcache</groupId>
 		   <artifactId>ehcache</artifactId>
        </dependency>
+       <dependency>
+		    <groupId>commons-validator</groupId>
+		    <artifactId>commons-validator</artifactId>
+		    <version>1.4.0</version>
```

Comment:
I thought this was already in core, sorry I didn't realize it, probably wasn't worth the effort.